config/params/fork_test.go

```diff
@@ -130,10 +130,10 @@ func TestNextForkData(t *testing.T) {
 			wantedEpoch:       cfg.BellatrixForkEpoch,
 		},
 		{
-			name:              "after last bpo - should be far future epoch and 0x00000000",
+			name:              "post last full fork, fulu bpo 1",
```

Comment:
@kasey I updated this test name based on feedback it gives the bpo 1 after setting